src/WebUI/WorkLoads/WorkloadDetails.tsx

```diff
@@ -187,6 +185,9 @@ export class WorkloadDetails extends BaseComponent<IWorkloadDetailsProperties, I
                 selectedPod: pods && pods.length > 0 ? pods[0] : null
             });
         }
+
+        this._adjustWorkloadColumnDetailsSize();
```

Comment:
logic of width computation

Reply:
removed this logic as we are using flex-grow and it is working perfectly alright.